File: android/interactable/src/main/java/com/wix/interactable/physics/PhysicsObject.java
package com.wix.interactable.physics;

import android.graphics.Point;

/**
 * Created by rotemm on 09/02/2017.
 */

public class PhysicsObject {

    Point velocity;
    float mass;

    public PhysicsObject() {
        this.velocity = new Point(0,0);
        this.mass = 1.0f;
    }

    public PhysicsObject(Point velocity, float mass) {
        this.velocity = velocity;
        this.mass = mass;
    }
}

File: ios/Interactable/PhysicsObject.m
//
//  PhysicsObject.m
//  Interactable
//
//  Created by Tal Kol on 2/4/17.
//  Copyright © 2017 Wix. All rights reserved.
//

#import "PhysicsObject.h"

@implementation PhysicsObject

- (instancetype)init
{
    if ((self = [super init]))
    {
        self.velocity = CGPointZero;
        self.mass = 1.0;
    }
    return self;
}

@end
